prometheus/input.tf

```diff
@@ -85,6 +86,24 @@ locals {
     alertable_postgres_instances = local.internal_pg_map
   }
 
+  app_alert_rules_variables = {
+    cfapps_dashboard_url = var.apps_dashboard_url
+    apps                 = local.app_map
+  }
+
+  app_map = [for instance, settings in var.alertable_apps : {
+    app_fullname                   = instance
+    monitoring_space_name          = split("/", instance)[0]
```

Comment:
We don't really need the space as the app name is unique and contains the environment name